From 5e0cfb715a74209f1af039198cf4b82488fea677 Mon Sep 17 00:00:00 2001
From: Fabien Spindler <Fabien.Spindler@inria.fr>
Date: Tue, 12 May 2015 10:56:07 +0200
Subject: [PATCH 3/3] Fix segfault on sparc, powerpc...

---
 src/tools/io/vpParseArgv.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tools/io/vpParseArgv.cpp b/src/tools/io/vpParseArgv.cpp
index a662226..6dd569f 100644
--- a/src/tools/io/vpParseArgv.cpp
+++ b/src/tools/io/vpParseArgv.cpp
@@ -173,7 +173,7 @@ vpParseArgv::parse(int *argcPtr, const char **argv, vpArgvInfo *argTable,
       infoPtr = matchPtr;
       switch (infoPtr->type) {
       case ARGV_CONSTANT:
-  *((long long *) infoPtr->dst) = 1;
+  *((int *) infoPtr->dst) = 1;
          break;
       case ARGV_INT:
          nargs = (uintptr_t) infoPtr->src;
-- 
2.1.4

