From 40d0e02748bb6c6eee1a8b57a4f1208f5ab934f9 Mon Sep 17 00:00:00 2001
From: Fabien Spindler <Fabien.Spindler@inria.fr>
Date: Wed, 15 Jul 2015 10:24:18 +0200
Subject: [PATCH 4/5] Turn off HTML_TIMESTAMP to be able to reproduce same
 build

---
 doc/config-doxygen.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/config-doxygen.in b/doc/config-doxygen.in
index 2a21e8e..fbc55b4 100644
--- a/doc/config-doxygen.in
+++ b/doc/config-doxygen.in
@@ -972,7 +972,7 @@ HTML_COLORSTYLE_GAMMA  = 80
 # page will contain the date and time when the page was generated. Setting
 # this to NO can help when comparing the output of multiple runs.
 
-HTML_TIMESTAMP         = YES
+HTML_TIMESTAMP         = NO
 
 # If the HTML_DYNAMIC_SECTIONS tag is set to YES then the generated HTML
 # documentation will contain sections that can be hidden and shown after the
-- 
2.5.1

