From 9ce80851b6e006667d1ef36e1fa61685f0834682 Mon Sep 17 00:00:00 2001
From: Fabien Spindler <Fabien.Spindler@inria.fr>
Date: Sat, 9 Jan 2016 15:26:08 +0100
Subject: [PATCH 6/6] Remove videoReader test

---
 example/video/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/example/video/CMakeLists.txt b/example/video/CMakeLists.txt
index 1f54cd3..2a5990f 100644
--- a/example/video/CMakeLists.txt
+++ b/example/video/CMakeLists.txt
@@ -42,7 +42,7 @@ cmake_minimum_required(VERSION 2.6)
 find_package(VISP REQUIRED visp_core visp_io visp_gui)
 
 set(example_cpp
-  videoReader.cpp
+#  videoReader.cpp
   imageSequenceReader.cpp
 )
 
-- 
2.6.4

